quantlib/test/test_rate_helpers.py

```diff
@@ -26,13 +26,19 @@ def test_create_deposit_rate_helper(self):
         deposit_day_counter = Actual365Fixed()
 
 
-        helper = DepositRateHelper(
+        helper1 = DepositRateHelper(
```

Comment:
I would suggest avoiding cryptic variable names. You could use `helper_from_quote` and `helper_from_float` in place of `helper1` and `helper2`